apps/files_versions/lib/Sabre/VersionFile.php

```diff
@@ -109,6 +110,13 @@ public function setLabel($label): bool {
 		}
 	}
 
+	public function getMetadataAuthor(): string {
```

Comment:
What do you think about making it generic here too?
```suggestion
	public function getMetadata($key string): string {
```